testing/file_test/file_test_base.h

```diff
@@ -148,12 +132,29 @@ class FileTestBase : public testing::Test {
   static auto TransformExpectation(int line_index, llvm::StringRef in)
       -> testing::Matcher<std::string>;
 
-  const std::filesystem::path* path_;
+  const std::filesystem::path path_;
+};
+
+struct FileTestFactory {
+  // The test fixture name.
+  const char* name;
+  // A factory function for tests.
```

Comment:
Super small nit -- a blank line between these maybe?

```suggestion

  // A factory function for tests.
```